exporters/pelorus/utils.py

```diff
@@ -198,3 +201,25 @@ def get_env_var(var_name: str, default_value: Optional[str] = None) -> Optional[
         return default_value
 
     return env_var
+
+
+def get_k8s_client():
+    k8s_client = None
+    try:
+        k8sconfig = config.new_client_from_config()
+        k8s_client = DynamicClient(k8sconfig)
+    except config.config_exception.ConfigException:
+        # Try load config from cluster
+        config.load_incluster_config()
+        file_namespace = open(
+            "/run/secrets/kubernetes.io/serviceaccount/namespace", "r"
+        )
+        if file_namespace.mode == "r":
+            namespace = file_namespace.read()
+            print("namespace: %s\n" % (namespace))
```

Comment:
I'm wondering why this was here in the first place-- just printing what namespace the service account is in? Can't that just be checked in OpenShift directly?

If we keep this, then it looks like the old code had some issues:

1. It's checking that the file's mode is `r`, when it just opened it with `r`.
2. The file never gets closed.

We can just replace this whole part with `namespace = Path("/run/secrets/kubernetes.io/serviceaccount/namespace").read_text()`